lib/core/ui/dashboard/home/home_live_card.dart

```diff
@@ -0,0 +1,210 @@
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:live_score/core/theme/app_theme.dart';
+import 'package:live_score/core/ui/dashboard/home/home_details_button.dart';
+import 'package:live_score/features/dashboard/cubit/dashboard_cubit.dart';
+
+class DashboardLiveCard extends StatelessWidget {
+  const DashboardLiveCard(
+      {super.key,
+      required this.leagueName,
+      required this.homeName,
+      required this.awayName,
+      required this.leagueLogo,
+      required this.homeLogo,
+      required this.awayLogo,
+      required this.homeScore,
+      required this.awayScore,
+      required this.time});
+  final String leagueName;
+  final String homeName;
+  final String awayName;
+  final String leagueLogo;
+  final String homeLogo;
+  final String awayLogo;
+  final int? homeScore;
+  final int? awayScore;
+  final int? time;
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocBuilder<DashboardCubit, DashboardState>(
+      builder: (context, state) {
+        if (state is Success) {
```

Comment:
This whole widget is a mess, too much happening here, if I were to hop on this project, I would look at this for 5 minutes before I could decipher what it actually  does. 

Please, extract the code to separate classes with widget inside.

Reply:
I wasn't sure how to extract it properly, with that i mean how to group certain widgets since the card was like a one big custom widget but I think it's fine after refractoring now.